virttest/virsh.py

```diff
@@ -4997,3 +4997,16 @@ def set_user_sshkeys(name, user, options="", **dargs):
     :return: CmdResult object.
     """
     return command("set-user-sshkeys %s %s %s" % (name, user, options), **dargs)
+
+
+def domdirtyrate_calc(name, options="", **dargs):
+    """
+    Calaulte vm's dirty page rate
```

Comment:
Hi @kylazhang ,The word "Calculate" is spelled incorrectly in this place, which will cause the CI check to fail. 
The error like this:
```
Run pylint --errors-only --disable=all --ignore=avocado-libs --enable=spelling --spelling-dict=en_US --spelling-private-dict-file=spell.ignore *
************* Module virttest.virsh
virttest/virsh.py:5008:0: C0402: Wrong spelling of a word 'Calaulte' in a docstring:
    Calaulte vm's dirty page rate
    ^^^^^^^^
Did you mean: ''Calculate''? (wrong-spelling-in-docstring)
Error: Process completed with exit code 16.
```